### PR TITLE
Note, Notebook, Methods

### PR DESCRIPTION
**Pull request by tufankaracam**
Note class created.
Notebook class created.
Both of classes implemented to main.
Methods updated for different types.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -142,6 +143,29 @@ def search(args, contacts: AddressBook):
     return contacts.search(args[0].strip())
 
 
+def add_note(args, notes: Notebook):
+    text = ' '.join(args)
+    return notes.create_note(text)
+
+
+def update_note(args, notes: Notebook):
+    *text, id = args
```
Comment (pavlomatchak) [inline main.py:152]: Wouldn't it be more intuitive to have id first?

File: notebook.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            self.data = {}
+
+    def save_records(self):
+        with open(self.filename, 'wb') as f:
+            pickle.dump(self.data, f)
+
+    def create_note(self, text: str):
+        id = self.__id
+        note = Note(id, text, datetime.now())
+        self[id] = note
```
Comment (pavlomatchak) [inline notebook.py:33]: self.data[id]?

Comment (tufankaracam) [reply notebook.py:33]: self[id] and self.data[id] , they are same :)

File: main.py (hunk 1 above)
Comment (tufankaracam) [reply main.py:152]: I will send new pull request, but first need apply this :)

**Review by pavlomatchak — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (pavlomatchak) [reply main.py:152]: Done

**Pull request merged**